src/chi/components/divider/divider.scss

```diff
@@ -0,0 +1,46 @@
+@import 'mixins';
+@import 'variables';
+
+.chi {
+  .a-divider {
+    border-top: 0.0625rem solid set-color(grey, 20);
+    margin: 0.5rem 0;
+
+    &.-dark {
```

Comment:
Let's remove `-dark` for now. I think long term it would make more sense to use border utilities to color dividers. We can solve that in a future release.